README.md

```diff
@@ -239,6 +239,23 @@ provides two options:
 This modality is an optional installation parameter and will default to the `Burnable` mode if not provided. However, this
 mode cannot be changed once the contract has been installed. The mode is set by passing a `u8` value to the `burn_mode` runtime argument.
 
+#### ReportingMode
+
+The `ReportingMode` modality dictates whether the contract supports retrieving token identifiers for a given token owner.
+Additionally, it also dictates whether receipts pointing to the various pages are also returned by the `mint` and `transfer`
+entrypoints. This modality provides two options:
+
+1. `NoReport`: The reporting and receipt functionality is not supported.
+2. `Report`: The reporting and receipt functionality is supported.
+
+| ReportingMode | u8  |
+|---------------|-----|
+| Report        | 0   |
+| NoReport      | 1   |
+
+This modality is an optional installation parameter and will default to the `NoReport` mode if not provided. However, this
+mode cannot be changed once the contract has been installed. The mode is set by passing a `u8` value to the `reporting_mode` runtime argument.
+
```

Comment:
let's rename this to `OwnerReverseLookupMode, and rename the variants to `Complete` and `None`...leaving the door open later to a non-binary choice like `TransferredOnly` or sth like that

Reply:
@EdHastingsCasperLabs I am going to go with `NoLookup` as opposed to `None`, just to keep it distinct from the `Option::None`

Reply:
> 

@ACStoneCL Please account for this readme PR, I think you and I have discussed the WHY, just pointing it out here AGAIN